### DISCUSSION TITLE
Possible to use w/ third-party UIs?

### DISCUSSION THREAD

**Discussion by aolko** [LostRuins/koboldcpp#832] [Q&A] [answered]
Is it possible to use koboldcpp with third-party UIs like https://github.com/vegu-ai/talemate ?

**Comment by LostRuins** [comment]
If they support the OpenAI compatible API, sure. So for this, probably yes.

**Reply by aolko** [reply, marked as answer]
checked yesterday, it is, indeed, possible, **however**, for endpoint **you have to use** `http://localhost:5001/v1` **and api key** `0000`
